pkg/cri/resource-manager/policy/builtin/static-pools/stp-policy.go

```diff
@@ -478,6 +490,18 @@ func (stp *stp) allocateStpResources(c cache.Container, cs stpContainerStatus) e
 
 	c.SetEnv(CmkEnvAssigned, cpuset)
 
+	// Advertise CPUs belonging to the infa pool
+	pool, ok = stp.conf.Pools[CmkPoolInfra]
+	if ok {
+		c.SetEnv(CmkEnvInfra, pool.cpuSet())
+	}
+
+	// Advertise CPUs belonging to the shared pool
+	pool, ok = stp.conf.Pools[CmkPoolShared]
+	if ok {
+		c.SetEnv(CmkEnvShared, pool.cpuSet())
+	}
```

Comment:
Would it make sense to advertise these also in the sh-syntax `resources file` ?

`cri-resource-manager` injects such a file into every `CreateContainer ` request as `/.cri-resmgr/resources.sh`. To add this data to it, you just need to change the currently empty `ExportResourceData()` to return the desired key-value pairs as a `map[string]string`.

Reply:
> Would it make sense to advertise these also in the sh-syntax `resources file` ?

Good point, I really don't know 😄  Could you remind me what is the intended use for the file?

In any case I think this should be handled in a separate issue/PR, I opened #370 for discussing this.

Reply:
The intended case is to provide an event-driven way (inotify) for (cri-resmgr-aware) containers to monitor and react to changes in their resource assignments.